charts/ccd-docker-user-profile-importer/templates/user-profile-importer.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if .Values.importer.userprofile.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-user-profile-importer"
```

Comment:
Make releasename templated with releasenameoverride and tpl support 
https://github.com/hmcts/chart-java/blob/master/java/templates/deployment.yaml#L5
https://github.com/hmcts/chart-java/blob/master/java/templates/_helpers.tpl#L83